src/components/main.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SectionWelcome from './sectionWelcome/sectionWelcome';
```

Comment:
компоненты называются с большой буквы
```suggestion
import SectionWelcome from './WelcomeSection/WelcomeSection';
```